pkg/symlink/fs.go

```diff
@@ -45,7 +45,7 @@ func FollowSymlinkInScope(path, root string) (string, error) {
 // would return "/foo/bar". If one makes /foo/bar a symlink to /baz subsequently, then "/foo/bar" should
 // no longer be considered safely contained in "/foo".
 func evalSymlinksInScope(path, root string) (string, error) {
-	root = filepath.Clean(root)
+	root = filepath.Clean(filepath.FromSlash(root))
```

Comment:
@jhowardmsft why not make this change in FollowSymlinkInScope instead? That way you could handle both `path` and `root` at the `IsAbs` level which would make it more intelligible imho.